### PR TITLE
Display error to the front-end

### LINKED ISSUE #65: Demo to display error messages
Following #60 ; low priority, but it'd be nice to show errors when they happen in the demo. To give an example on how to handle them. Add it to the docs, also.

### PR DESCRIPTION
**Pull request by brunobord**
closes #65
- prints status and error objects in the console,
- display error messages in a DHTML div
- amend documentation with a literal example

### CODE REVIEW

**Comment by brunobord** [comment]
@amessinger can you have a quick look at my JS, please?

File: demo/static/js/demo/selectize.js [hunk 1]
```diff
@@ -1,4 +1,20 @@
 $(document).ready(function() {
+
+    function error_callback(resp) {
```
Comment (wo0dyn) [inline demo/static/js/demo/selectize.js]: I would use cameCase for JS functions/methods.

``` js
function errorCallback(resp) {
```

File: docs/error-handling.rst [hunk 2]
```diff
@@ -21,3 +21,13 @@ This error payload format is inspired by `JSON API <http://jsonapi.org/format/#e
...
+
+Display errors on the front-end side
+------------------------------------
+
+You may browse the ``selectize.js`` file, that shows an example on how to handle errors and use them in your user interface:
+
+.. literalinclude:: ../demo/static/js/demo/selectize.js
+    :language: javascript
+    :lines: 3-16
+    :dedent: 4
```
Comment (wo0dyn) [inline docs/error-handling.rst:33]: :+1:

**Review by wo0dyn — COMMENTED** [review]
I noticed a nitpick but LGTM.

**Comment by brunobord** [comment]
will fix the JS if needed.

**Pull request merged**